.circleci/config.yml

```diff
@@ -797,16 +891,12 @@ workflows:
   nightly:
     triggers:
       - schedule:
-          cron: "0 22 * * *"
+          cron: "0 22 * * *" # daily around 6 PM EST
           filters:
             branches:
               only:
                 - develop
     jobs:
-      - playwright-e2e-test:
```

Comment:
Should we bring this back so that the playwright test run every couple of hours? Granted I'm not sure if the exact amount of hours has been decided (going off the meeting agenda).

Reply:
It will be a different ticket to work on the nightly addition. Right now, job is broken, fails on every nightly builds.